Makefile

```diff
@@ -32,8 +33,8 @@ all-srcs:
 
 .DEFAULT_GOAL := addlicense-fmt-vet-lint-test
```

Comment:
Shouldn't this be updated to `addlicense-fmt-vet-lint-goimports-test`?

Reply:
Good catch, fixed.